app/users/services/ClientService.py

```diff
@@ -118,7 +122,7 @@ def client_send_verification_code(cls, request):
                     status=response_data.get("status_code"),
                 )
 
-            sid = TwilioVerifyService.send_verification_code(phone_number)
+            sid = cls.sms_verification_service.send_verification_code(phone_number)
```

Comment:
how are you handling the exception when there's an error from twilio
and where you are saving the details for the sms verification